packages/core/src/components/Form.tsx

```diff
@@ -815,6 +829,7 @@ export default class Form<
           disabled={disabled}
           readonly={readonly}
         />
+
```

Comment:
Something like:
```suggestion
// Add SUBMIT_BTN_OPTIONS_KEY and UI_OPTIONS_KEY to the import for @rjsf/utils
let { [SUBMIT_BTN_OPTIONS_KEY]: submitOptions = {} } = getUiOptions<T, S, F>(uiSchema);
if (disabled) {
  submitOptions = { ...submitOptions, submitButtonProps: { ...submitOptions.submitButtonProps, disable: true } };
}
const submitUiSchema = { [UI_OPTIONS_KEY]: { [SUBMIT_BTN_OPTIONS_KEY]: submitOptions } };
```

Then pass 
```
        {children ? children : <SubmitButton uiSchema={submitUiSchema} registry={registry} />}
```